### PR TITLE
Improve SOFB Device

### PR DESCRIPTION
**Pull request by fernandohds564**
- Improve `devices.SOFB` to correctly deal with SinglePass and MultiTurn trajectories.

### CODE REVIEW

File: siriuspy/siriuspy/devices/device.py [hunk 1]
```diff
@@ -91,14 +91,15 @@ def values(self):
 
     def __getitem__(self, propty):
         """Return value of property."""
-        pvobj = self._pvs[propty]
-        value = pvobj.get()
-        return value
+        pvobj = self._pvs.get(propty)
```
Comment (xresende) [inline siriuspy/siriuspy/devices/device.py:94]: pyling `inconsistent-return-statement`

File: siriuspy/siriuspy/devices/device.py [hunk 2]
```diff
@@ -91,14 +91,15 @@ def values(self):
...
+        if pvobj is not None:
+            return pvobj.get()
 
     def __setitem__(self, propty, value):
```
Comment (xresende) [inline siriuspy/siriuspy/devices/device.py:98]: same here

File: siriuspy/siriuspy/devices/device.py (hunk 1 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/devices/device.py:94]: I don't understand. I don't get these errors.
What is it complaining about?

Comment (fernandohds564) [reply siriuspy/siriuspy/devices/device.py]: done!

File: siriuspy/siriuspy/devices/device.py (hunk 2 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/devices/device.py]: done!

**Pull request merged**